what4/src/What4/BaseTypes.hs

```diff
@@ -290,19 +290,19 @@ instance Hashable (StringInfoRepr si) where
   hashWithSalt = $(structuralHashWithSalt [t|StringInfoRepr|] [])
 
 instance Pretty (BaseTypeRepr bt) where
-  pretty = text . show
+  pretty = pretty . show
```

Comment:
I think this changes the formatting by quoting the strings.  In this, and similar places, I think we should use the `viaShow` method you mention from https://github.com/GaloisInc/crucible/pull/586 instead.

Reply:
Agreed. I'll rewrite the code again to avoid ever using `pretty . show`, favoring `viaShow` instead.